src/Heading/index.js

```diff
@@ -49,6 +76,8 @@ const createHeading = (
     ? `margin-top: var(--spectrum-global-dimension-size-500); `
     : ''
 
+  const innerCss = generateCss(isHeading1, isHeading2)
```

Comment:
why not just pass in `level` instead and have `generateCss` switch on it? or is there a reason why we have to use the `isHeading*` bools?

Reply:
I think the confusing part for me when reading the code is more that it's either one or the other. That being said, this is more a nitpick.

Reply:
That being said if you _do_ go that route and fiddle w/ `level`, would it make sense to do a boundary/index check on `level`? Asking just because of the `spectrum-Heading--size${headingSizes[level - 1]}` below.

If you do something with that it would be great to have a test.

Reply:
It's not either one or the other, there is css for h1, different css for h2 and a third option which covers h3-h6. So I can use level and change it to a switch statement but it will read very much the same.

We don't do a boundary check because `createHeading` is not exported outside of this file and you can't over-ride the level property. So you can only create h1-h6 with the Heading component which means you'll never get an array out of bounds error.